infra/aws/terraform/prow-build-cluster/resources/monitoring/grafana/config.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: monitoring
+  name: grafana
+data:
+  grafana.ini: |-
```

Comment:
Maybe we want to take some stuff from https://github.com/kubernetes/test-infra/blob/26ca71bf2d66347b7d17f28d05ac4d24d1632154/config/prow/cluster/monitoring/grafana_configmaps.yaml#L59

Reply:
Looks good. Let me try it.